spec/mc_settings_spec.rb

```diff
@@ -11,6 +11,16 @@
           :local => true)
     end
 
+    it 'should access via ... something' do
+      expect(Setting.two[:three]).to eq(5)
```

Comment:
A number of these tests already exist [here](https://github.com/stitchfix/fittings/blob/main/spec/mc_settings_spec.rb#L59-L66). Can we consolidate them?

Reply:
Thanks, fixed in 6dcdf60f17f34cb191c8eaeafef06d6263d5a4f9.